### ISSUE TITLE
Improve the settings of `AnnubesEnv`

### ISSUE THREAD

**Issue by gcroci2** [neurogym/neurogym#63]
**[<img src="https://avatars.githubusercontent.com/u/55382553?s=17&v=4" width="17" height="17"> @gcroci2](https://github.com/gcroci2)** cloned issue [ANNUBS/neurogym#68](https://github.com/ANNUBS/neurogym/issues/68) on 2024-11-18: 

> We need to organize in a clearer way the setting of `AnnubesEnv` (and likely, all the envs). Dataclasses did not convince me completely (see the discussion in https://github.com/ANNUBS/annubes/issues/36) and I think we should look for which are the best options around. This is for a later stage, when the entire training pipeline will be in place.

**Comment by cantordust** [comment]
@gcroci2 @DaniBodor I will take this on if you have no objections. I designed a solution for this already while working on the weight extraction & tracking issue, and I realised that it can be used for configuring environments in a very generic way (quite in line with what @DaniBodor mentioned about using mixins).

**Comment by cantordust** [comment]
@DaniBodor I cannot seem to be able to change the status of issues - perhaps I don't have the permissions? I'd like to move this to "In progress".

**Comment by DaniBodor** [comment]
> @gcroci2 @DaniBodor I cannot seem to be able to change the status of issues - perhaps I don't have the permissions? I'd like to move this to "In progress".

I made you an admin (I just double checked), so you should have the same rights as us. I usually drag and drop the issue from one column to another in the project board. Alternatively, inside the issue on the right, the status has a drop down menu, where you can change it. Does neither of this work for you?

**Comment by gcroci2** [comment]
> Thank you for looking into this. I tried both of those methods before I posted the comment, but indeed I couldn't drag and drop (and still can't), and the status dropdown was (and still is) greyed out. Weird! I even logged out and back in - no change.🤔

I moved this issue to "In Progress" now. Indeed it's weird that it's not working for you! We can look at it together after the holidays if the issue persists.

**Comment by cantordust** [comment]
I suspect that I might have to be a member of the organisation - right now I think I'm only a member of the project but not the org...

**Comment by gcroci2** [comment]
> I suspect that I might have to be a member of the organisation - right now I think I'm only a member of the project but not the org...

That makes sense! I've just invited you

**Comment by cantordust** [comment]
Yes, that was it! I can change the status now. Thank you for resolving this!🖖🤓
